lochness/email/bootdey_template.html

```diff
@@ -0,0 +1,78 @@
+<html>
```

Comment:
Can you send me the template HTML for this?

Reply:
https://www.bootdey.com/snippets/view/bs4-Email-alert

Reply:
Sorry, do you mean this one?
https://www.bootdey.com/snippets/view/bs4-Email-alert#html

Reply:
yes

Reply:
I don't think that is right. That HTML looks like this:

![image](https://user-images.githubusercontent.com/35086881/149843096-e16dddda-c56b-4421-adb9-577ec60ab2bd.png)

Reply:
Just attaching `bootdey_template.html` that Kevin coded for future reference:

![image](https://user-images.githubusercontent.com/35086881/149844164-f7b09ebf-1286-4bce-a50f-f3f375c75ba3.png)

Reply:
oh my bad. It's here https://github.com/AMP-SCZ/lochness/blob/kcho/email/lochness/email/bootdey_template.html